atomate/vasp/firetasks/run_calc.py

```diff
@@ -147,6 +148,11 @@ def run_task(self, fw_spec):
         elif job_type == "full_opt_run":
             jobs = VaspJob.full_opt_run(vasp_cmd, auto_npar=auto_npar, ediffg=self.get("ediffg"),
                                         max_steps=5, half_kpts_first_relax=False)
+        elif job_type == "neb":
```

Comment:
Using a half KPOINTS is based on our experience in NEB calculations, and we would claim that most of them need to do so to achieve a reasonable convergence speed. To keep the consistency between the settings in atomate, we have modified **VaspNEBJob** arguments and now only "auto_npar" and "gamma_vasp_cmd" are allowed to change. So we no longer have conflict between the optional parameters. Alternatively, we have added two optional arguments user_kpoints_settings and additional_cust_args to every fireworks, so that the user can change k-mesh and custodian settings based on their own preference.